packages/tenant-process/src/services/tenantService.ts

```diff
@@ -352,4 +450,70 @@ export function tenantServiceBuilder(
     },
   };
 }
+
+async function assignCertifiedAttribute({
+  targetTenant,
+  certifiedTenantAttribute,
+  attribute,
+  organizationId,
```

Comment:
Where is `organizationId` used?

Reply:
to throw `certifiedAttributeAlreadyAssigned(attribute.id, organizationId)`